.github/workflows/visual-regression.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     steps:
 
     - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
[Just a question] Does this find the head branch of the forked repository?

Reply:
Yes, `github.event.pull_request` works for the PRs from forked repository. Please take a look at [this PR](https://github.com/optuna/optuna-visualization-regression-tests/pull/8#issuecomment-1488068308). It was created from @eukaryo 's repository to optuna/optuna-visualization-regression-tests, and the PR head was checked out as expected.